lib/graphql/analysis/ast/field_usage.rb

```diff
@@ -48,7 +48,7 @@ def extract_deprecated_arguments(argument_values)
               argument_type = argument_type.of_type
             end
 
-            if argument_type.kind.input_object?
+            if argument_type.kind.input_object? && argument.value.respond_to?(:arguments) # Skip if value is not a GraphQL::Schema::InputObject
```

Comment:
We are running into the same issue with the graphql-ruby upgrade and this problem also happens below on line 60.

That is the case where the argument is a list of input objects. The call to `value.arguments` there fails too.